.gitattributes

```diff
@@ -0,0 +1 @@
+*	-text
```

Comment:
The latter. The only time lint is run on this repo is as part of the tests.

Pretty sure all you need for AppVeyor is `init: git config --global core.autocrlf true` though I could be wrong.